lib/routes/forklog/index.ts

```diff
@@ -0,0 +1,73 @@
+import { Route } from '@/types';
+import { parseDate } from '@/utils/parse-date';
+import got from '@/utils/got';
+import timezone from '@/utils/timezone';
+
+export const route: Route = {
+    path: '/news',
+    categories: ['finance'],
+    example: '/forklog/news',
+    radar: [
+        {
+            source: ['forklog.com/news'],
+            target: '/news',
+        },
+    ],
+    name: 'Forklog – Новости',
```

Comment:
```suggestion
    name: 'Новости',
```
https://github.com/DIYgod/RSSHub/pull/18477#discussion_r1975713745 is not fixed

Reply:
Sorry, replaced in the wrong place. Now fixed though